bot.py

```diff
@@ -112,6 +112,64 @@ async def list(ctx):
     )
 
 
+@bot.command()
+async def add(ctx):
+    author = ctx.message.author.id
+    await ctx.message.channel.send(
+        embed = Embed.from_dict(
+            {
+                "fields": [
+                    {
+                        "name": "Enter items to add to personal inventory:",
+                        "value": "\u200b"
+                    }
+                ]
+            }
+        )
+    )
+    add_to = await bot.wait_for('message', timeout = 60 )
```

Comment:
```suggestion
    add_to = await bot.wait_for("message", timeout=60 )
```

See previous comment about spacing. Use `"` instead of `'` for strings.